AUDIT.md

```
@@ -0,0 +1,384 @@
+## Issue #1: Why /ui? Confusing Routing Structure
+
+### Current Routing (`src/index.ts`)
+
+```typescript
+app.get("/", (c) => c.redirect("/ui"));  // Root redirects to /ui
+app.route("/ui", uiRouter);              // All UI under /ui prefix
+```
+
+### Current UI Routes (`src/routes/ui.tsx`)
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)